src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -149,11 +127,11 @@ public function testValidateIfChildWithValidConstraint()
         $this->assertNoViolation();
     }
 
-    public function testDontValidateIfParentWithoutCascadeValidation()
+    public function testDontValidateIfParentWithoutValidConstraint()
```

Comment:
I'd appreciate if you could check the other test cases whether any of them duplicates this one. If that's the case, we can safely remove this. This was mainly designed in order to test the special treatment of the `cascade_validation` option.